### PR TITLE
Splited code of components in layouts

### LINKED ISSUE #739: Code splitting in layout components
@AJIL-PAUL _a

Please do basic level components splitting in the layout files. Create folders if the file is split into 2 or more files.

**No props or logic of components should be changed.**

### PR DESCRIPTION
**Pull request by AJIL-PAUL**
Fixes #739 

Splitted codes in the following components under layouts:
```
- Sidebar
    |_ index.js
    |_ FeaturedTooltip.js
    |_ NavIconWrapper.js
    |_ NeetoLogo.js
    |_ ProfileSection.js
    |_ ProfileSectionTooltip.js
    |_ Footer
         |_ index.js
         |_ AppSwitcher.js
         |_ ChangeLog.js
    |_constant.js

- AppSwitcher
    |_ index.js
    |_ AppLink.js
    |_ Body.js

```

@amaldinesh7 _a Please review

### CODE REVIEW

File: lib/components/layouts/Sidebar/index.js [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+        showAppSwitcher={showAppSwitcher}
+        changelogProps={changelogProps}
+        toggleCollapse={toggleCollapse}
+        onAppSwitcherToggle={onAppSwitcherToggle}
+      />
+    </motion.div>
+  );
+};
+
+Sidebar.propTypes = {
```
Comment (amaldinesh7) [inline lib/components/layouts/Sidebar/index.js:194]: Can we move this proptypes object into constants?

File: lib/components/layouts/AppSwitcher/index.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+              )}
+            </motion.div>
+          </Backdrop>
+        )}
+      </AnimatePresence>
+    </Portal>
+  );
+};
+
+AppSwitcher.propTypes = {
```
Comment (amaldinesh7) [inline lib/components/layouts/AppSwitcher/index.js:118]: Move to constants

File: lib/components/layouts/AppSwitcher/index.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+import Body from "./Body";
+import { Portal, Backdrop } from "../../../atoms";
+import {
+  SWITCHER_SIZE,
+  BLUR_INITIAL,
+  BLUR_FINAL,
+  TRANSITION,
+} from "../../../constants/overlay";
+
+const sizes = { lg: "lg", sm: "sm" };
```
Comment (amaldinesh7) [inline lib/components/layouts/AppSwitcher/index.js:16]: move to constants

File: lib/components/layouts/AppSwitcher/index.js [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+const noop = () => {};
+
+const envs = {
```
Comment (amaldinesh7) [inline lib/components/layouts/AppSwitcher/index.js:20]: move to constants

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@AJIL-PAUL _a Please check the comments

**Comment by AJIL-PAUL** [comment]
@amaldinesh7 _a Updated based on comments. Please review

**Pull request merged**